docs/databases/swift-kuery.html

```diff
@@ -221,7 +218,7 @@ <h2 class="heading-2"><span class="blue-text">Step 2b:</span> Create a Postgres
             <p>Now we're ready to connect to our database from our Kitura server.</p>
             <div class="underline"></div>
             <h2 class="heading-2"><span class="blue-text">Step 2c:</span> Create a connection to a PostgreSQL database</h2>
-            <p>Add <a href="https://github.com/IBM-Swift/Swift-Kuery" target="_blank">Swift-Kuery</a> and <a href="https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL" target="_blank">SwiftKueryPostgreSQL</a> to the dependencies in the `Package.swift` file. </p>
+            <p>Add <a href="https://github.com/IBM-Swift/Swift-Kuery" target="_blank">Swift-Kuery</a> and <a href="https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL#add-dependencies" target="_blank">SwiftKueryPostgreSQL</a> to the dependencies in the `Package.swift` file. </p>
```

Comment:
It's confusing that Swift-Kuery doesn't link to an add dependencies section in the README. We should be consistent about linking to that section (and making one in the README if required).

In addition we should align the Swift-Kuery README and the guide. Currently in the guide we're importing both packages explicitly to ensure you have the most recent version of each, however that's not what the Swift-Kuery README suggests users do. Which are we recommending to users?

Reply:
We agreed that you should explicitly import Swift-Kuery and should update the Kuery guide to reflect this.